packages/connect/src/ExtensionProvider/ExtensionProvider.ts

```diff
@@ -165,6 +156,7 @@ export class ExtensionProvider implements ProviderInterface {
     }
     window.postMessage(initData, '*');
     window.addEventListener('message', ({data}) => {
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
```

Comment:
You should follow the guidelines on how to correctly lint this instead of disabling linter.
For no-unsafe-member-access rule there are some correct guidelines here:
https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-unsafe-member-access.md

Same for across PR where disables take place

Reply:
I think I cleared this up in chat yesterday. Can you approve this now?

Reply:
I opened #189

Reply:
@raoulmillais - Yes we cleared this up. Comment was before our talk